api/proto/teleport/legacy/types/types.proto

```diff
@@ -208,6 +208,9 @@ message DatabaseServerV3 {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "status"
   ];
+  // The advertized scope of the server. A server's scope can not change once assigned, so
+  // heartbeats must include a scope value matching the one declared in the hello message.
```

Comment:
Echoing Tiago's comment in `UpstreamInventoryHello`, these comments are picked up by docs generators so the description in the `FooServer` messages should probably be reworded to avoid mentioning internals if possible.